packages/units/src/getBoundRect.ts

```diff
@@ -130,7 +143,13 @@ export function getCorners(angle, x, y, width, height) {
   };
 }
 
-function getCorner(pivotX, pivotY, cornerX, cornerY, angle) {
```

Comment:
<!-- propose improvements -->
**💬 suggestion:**
Looks like the utils in this file don't have return types, maybe add? Or is it purposefully so?

Reply:
They're implicit.

Right now we do not enforce explicit return types, though [there's a lint rule for that](https://typescript-eslint.io/rules/explicit-function-return-type/).